### PR TITLE
docs: and Carbon champion names

### LINKED ISSUE #3753: Website docs: Champions section
We plan to display names of Carbon champions on the /contribute/get-started/ page. Here's why:

Carbon is IBM’s open source design system for products and digital experiences and the system consists of working code, design tools and resources, human interface guidelines, and a vibrant community of contributors. Because we rely so heavily on contributors, we have a Carbon champion program where we celebrate folks that have made a significant impact on the design system. Right now, we simply post in slack or mention these names in quarterly calls. But our hypothesis is that by displaying their names on the website, winners will feel more gratification for their work and this will also encourage others to contribute.

This is the [prototype of our new contribution page](https://carbondesignsystem-git-fork-aubrey-fd4a89-carbon-design-system.vercel.app/contributing/get-started/#carbon-champions) where we will display carbon champion names.

- [x] Draft consent form
- [x] Get form reviewed by Bill Chan in legal
- [x] Add form to [airtable](https://airtable.com/appOVMXJWJgb6dwhX/tblHFlEESn3dqumFn/viw8ZHWRoiQqZvLgm?blocks=hide)
- [x] Collect signature
- [ ] Update contribution/get-started page

### PR DESCRIPTION
**Pull request by aubrey-oneal**
Closes #3753 

#### Changelog

**New**

- Add Carbon champion names (only those who have filled out the consent form)

**Removed**

- Nomination form (we can't have this because it collects PI)
- Contribution Gallery section with Airtable iframe

### CODE REVIEW

File: src/pages/contributing/get-started.mdx [hunk 1]
```diff
@@ -34,28 +33,43 @@ stop on by. We'll help you find an appropriate first contribution depending on
...
+
+**Valerie Garza**<br /> Gantt chart<br /><br /> **Michael Gower**<br />
+Accessibility<br /><br /> **Mark Judy**<br /> Typescript<br /><br /> **Chris
+Knabe**<br /> React Testing Library
+
+</Column>
+
+<Column className="one" colSm={2} colMd={2} colLg={3}>
+
+**Marcin Lewandowsk**i<br /> Typescript<br /><br /> **Jessica Lin**<br />
```
Comment (andreancardona) [inline src/pages/contributing/get-started.mdx:60]:
```suggestion
**Marcin Lewandowski**<br /> Typescript<br /><br /> **Jessica Lin**<br />
```

File: src/pages/contributing/get-started.mdx [hunk 2]
```diff
@@ -34,28 +33,43 @@ stop on by. We'll help you find an appropriate first contribution depending on
...
+Accessibility<br /><br /> **Amaya Mali**<br /> Gantt chart<br /><br /> **Tom
+Roach**<br /> Community support<br /><br />
+
+</Column>
+
+<Column className="one" colSm={2} colMd={2} colLg={3}>
+
+**Eugene To **<br /> Figma<br /><br /> **Shankar Venkatachalam** <br /> Create
+React App<br /><br /> **Colleen Yates**<br /> llustration Guild
```
Comment (andreancardona) [inline src/pages/contributing/get-started.mdx:69]:
```suggestion
React App<br /><br /> **Colleen Yates**<br /> Illustration Guild
```

**Review by andreancardona — COMMENTED** [review]
a few small things otherwise looks good!

**Review by laurenmrice — APPROVED** [review]
This looks good to me pending the typos Andrea mentioned and Jeannie's review of the Carbon Champions layout.

File: src/pages/contributing/get-started.mdx (hunk 1 above)
Comment (aubrey-oneal) [reply src/pages/contributing/get-started.mdx]: Good catch! Just fixed this.

File: src/pages/contributing/get-started.mdx (hunk 2 above)
Comment (aubrey-oneal) [reply src/pages/contributing/get-started.mdx]: Good catch! Just fixed this.

**Review by jeanservaas — APPROVED** [review]
I like the new champions look!

**Comment by jeanservaas** [comment]
@aubrey-oneal one thing...

For Shankar, since you don't have a verb anywhere else, maybe I'd remove the "Create" for Shankar

image

**Review by andreancardona — APPROVED** [review]

**Pull request merged**